test/Microsoft.DotNet.ProjectModel.Tests/GivenThatIWantToLoadAProjectJsonFile.cs

```diff
@@ -747,7 +747,7 @@ public void It_adds_the_dependency_when_the_version_is_set_directly()
             dependency.Type.Should().Be(LibraryDependencyType.Default);
             dependency.SourceFilePath.Should().Be(ProjectFilePath);
             dependency.SourceLine.Should().Be(3);
-            dependency.SourceColumn.Should().Be(31);
+            dependency.SourceColumn.Should().Be(30);
         }
```

Comment:
It's part of a bug fix that happened in Json.NET, I debugged through the code and we grab that value directly from Json.NET. The new position was just one space away or one " away.